addons/l10n_hu_edi/data/uom.uom.csv

```diff
@@ -3,8 +3,6 @@
 "uom.product_uom_day","DAY"
 "uom.product_uom_hour","HOUR"
 "uom.product_uom_meter","METER"
-"uom.product_uom_km","KILOMETER"
 "uom.product_uom_litre","LITRE"
-"uom.product_uom_cubic_meter","CUBIC_METER"
```

Comment:
err, removing these from the localizations seems to contradict other selection fields within them:

https://github.com/odoo/odoo/blob/d6c3bf11a4a2a333268f78f40d8f92a37d58d443/addons/l10n_hu_edi/models/uom_uom.py#L12-L21

so another reason to not remove any of them